### DISCUSSION TITLE
N64 crashing

### DISCUSSION THREAD

**Discussion by GrangeSec** [darthcloud/BlueRetro#625] [Q&A]
So I'm attempting to use a single controller input for now. Will expand to 4 once I get the proof of concept working. 

I'm using an official ESP32 board. I have connected all 3 controller wires. I have verified the GND, DATA, and 3V wires are all correct with a multimeter. However when I go to connect to the N64, it crashes. Power light will flash then go black. I won't be able to turn the n64 back on for a few minutes. 

What have I done wrong and how can I fix it? I've tried a search for similar discussion. The only relevant thing I found was someone saying they needed to use the USB cable. I tried that too. Same results. 

Blue retro 1.8.3

**Comment by darthcloud** [comment]
You are shorting your n64 3.3v and Gnd and are blowing the polyfuse.

Check your connection

**Reply by GrangeSec** [reply]
Interesting. I've checked using a multimeter 2x. :/ 

I'll check again.

**Comment by GrangeSec** [comment]
FYI, I got it working. I retried multiple times. It just didn't work. I'm not sure why, but once I removed the breakout board and soldered directly it worked. 🤷‍♂️

Same connections, same layout, just without the extra board. I looked at the breakout board and it looks fine.

KeeYees 2pcs ESP32 Breakout Board GPIO 1 into 2 for 38PIN No Mounting Hole Version ESP32 Development Board https://a.co/d/fpMOL6n
